packages/destination-actions/src/destinations/fullstory/__tests__/request-params.test.ts

```diff
@@ -0,0 +1,74 @@
+import { Settings } from '../generated-types'
+import { listOperationsRequestParams, setUserPropertiesRequestParams, deleteUserRequestParams } from '../request-params'
+import { forEachDataRegion, anonymousId, displayName, email, userId, apiKey } from './fullstory.test'
+
+const forEachDataRegionPlusFakeValues = (
+  callback: (settings: Settings, baseUrl: string, isValidRegion: boolean) => void
+) => {
+  forEachDataRegion((settings, baseUrl) => callback(settings, baseUrl, true))
+  ;['', 'fake'].forEach((fakeRegion) => callback({ apiKey, region: fakeRegion }, 'fake-base-url', false))
```

Comment:
me, tweaking a little bit :P 

Why leading semi-colon. Is this how the kids a writing typeScript these days?

I'm here to learn.

Reply:
> me, tweaking a little bit :P

You and I both! 😆

Segment's action destinations prettier config [omits semicolons at the end of lines](https://github.com/cowpaths/segmentio-action-destinations/blob/main/package.json#L77). Omitting the semicolon in this case treats the array on line 9 as an index into the result of `forEachDataRegion` (which is trying to index into `void`). I can work around this and make it a little less unsightly by extracting the fake regions into an array variable. WDYT?

```
  const fakeDataRegions = ['', 'fake']
  forEachDataRegion((settings, baseUrl) => callback(settings, baseUrl, true))
  fakeDataRegions.forEach((fakeRegion) => callback({ apiKey, region: fakeRegion }, 'fake-base-url', false))
```

Reply:
ah, yeah. I see it now - thank you. 

I think `const fakeDataRegions = ['', 'fake']` is much more readable - go ahead and refactor to use that statement.